kobo/apps/openrosa/libs/data/tests/test_tools.py

```diff
@@ -38,7 +38,7 @@ def test_get_form_submissions_grouped_by_field(self, mock_time):
     @patch('kobo.apps.openrosa.apps.logger.models.instance.submission_time')
     def test_get_form_submissions_grouped_by_field_datetime_to_date(
             self, mock_time):
-        now = datetime(2014, 1, 1)
+        now = datetime.now()
```

Comment:
Why did you change the date? I think you should keep the hard-coded value.

Reply:
I changed it to datetime.now() because we now store the current timestamp immediately upon the model being instantiated using an abstract model. Previously, date_created was not automatically being stored; instead, the value of submission_time was being stored in the date_created field. However, I have updated the test case by adding a mock datetime for date_created and by keeping datetime(2014, 1, 1).